README.md

```diff
@@ -1002,13 +1003,16 @@ There are some key subtleties you’re likely to come across in the way
   confusing, as it can be computed
   [more than one way](http://www.mattbartus.com/option-grants-fully-diluted-or-issued-and-outstanding/).
   Typically, people count shares in two ways: *issued and outstanding* and *fully diluted*.
-- 🄳 **Issued and outstanding** refers to the number of shares actually issued by the company
-  to shareholders, and does not include shares that others may have an option to purchase.
-- 🄳 **Fully diluted** refers to all of the shares that have been issued, all of the shares
-  that have been set aside in a stock incentive plan, and all of the shares that could be
-  issued if all convertible securities (such as outstanding warrants) were exercised.
-  A key difference is that this total will include all the shares in the employee option
-  pool that are reserved but not yet issued to employees.
+- 🄳 **Issued and outstanding** refers to the number of shares actually issued by a company to
+  shareholders, and does not include shares that others may have an option to purchase.
+- 🄳 **Fully diluted** refers to all of the shares that a company has issued, all of the
+  shares that have been set aside in a stock incentive plan, and all of the shares that
+  could be issued if all convertible securities (such as outstanding warrants) were
+  exercised.
+
+A key difference between fully diluted shares and shares issued and outstanding is that
+the total of fully diluted shares will include all the shares in the employee option pool
+that are reserved for but *not yet issued to* employees.
```

Comment:
remove extraneous "for"

Reply:
I wouldn't consider that extraneous. They are reserved for employees, but not yet issues to them. No?

Reply:
Yes, technically, but I'm not sure it adds clarity (I actually thought it was a typo). The relevant point in *this* context is that the pool is reserved, not that it's for employees. Also, the unused parts of the pool are not guaranteed to go to employees; it can get reallocated later, a point that probably should be mentioned in the pool definition. https://www.alleywatch.com/2014/01/vc-101-what-happens-to-the-employee-option-pool-after-an-acquisition/